Change the call + FAQs (#595)
ropensci/roweb3: content/champions/programdetails_2023.md

Author: I want to have all the different version because I think we will end up with a specific web page for the champion program if we continue with it after this two first cohort. So I will create the 2023-2024 timeline :-)

+++
title = "Program Details - Champions"
description = "If you want to be one of our champions here you can find goals, elegibility, benefits, and expectations for our program"
tags = ["champions"]
+++

{{< figure src = "/images/volunteer/peep4.png" width = "300" alt = "A person with his dog working on a notebook." class = "pull-right" >}}

## Program goals

> After the inaugural year of the _rOpenSci Champions Program_ we open the second call!!

Reviewer: ```suggestion
We're excited to open our second call for the _rOpenSci Champions Program_!
```

> The program aims to support R developers, from around the world, and who identify themselves as members of a group that is systematically excluded from the open software community to:

Reviewer: ```suggestion
The program aims to support R developers from around the world who identify themselves as members of groups which are systematically excluded from the open software community.

Through this program, Champions will:
```

* Gain the skills to develop their own R package or;
* Gain confidence in submitting an R package for peer review or become a reviewer.

> * Learn how to lead a workshop or other event about package development or software peer review -  or another rOpenSci activity - in their local community.

Reviewer: ```suggestion
* Learn how to lead a workshop, event, or other rOpenSci activity on the topic of package development or software peer review in their local community.
```

* Connect with a supportive network of peers and mentors within the R ecosystem.
* Provide feedback to help rOpenSci to continue to host relevant, welcoming activities for R developers.
* Gain transferable skills and experiences that are sought after for careers in open source software.

> We are looking six to nine Champions for this second group. We expect to co-create the rOpenSci Champion program with the selected applicants and influence future editions with their experience and feedback.

Reviewer: ```suggestion
We are looking six to nine Champions for this second group. We expect to co-create this iteration of the rOpenSci Champions Program with the selected applicants and refine future editions with their experience and feedback.
```

We look forward to receiving many applications and are committed to directing non-selected applicants towards other ways in which they can contribute to rOpenSci.

## Eligibility

{{< figure src = "/images/volunteer/peep3.png" width = "300" alt = "Person holding a butterfly nest." class = "pull-right" >}}

> We are looking for people that work on STEAM fields, from anywhere in the world, who identify with the above and who:

Reviewer: ```suggestion
We are looking for people who work in STEAM fields, from anywhere in the world, who identify with the above and who:
```

* Already have development knowledge with the R language and a solid idea for an R package they’d like to develop, or;

> * Have already developed an R package that is in [rOpenSci scope](https://devguide.ropensci.org/policies.html?q=scope#aims-and-scope), and want to have it [peer reviewed](/software-review/) through rOpenSci.

Reviewer: ```suggestion
* Have already developed an R package that is within [rOpenSci scope](https://devguide.ropensci.org/policies.html?q=scope#aims-and-scope), and want to have it [peer reviewed](/software-review/) through rOpenSci, or;
```

> * Want to become a [reviewer](https://devguide.ropensci.org/reviewerguide.html) for rOpenSci.

Reviewer: ```suggestion
* Want to become a [reviewer](https://devguide.ropensci.org/reviewerguide.html) for rOpenSci, and;
```

* Have basic English communication skills (written and verbal).

## About the program

The rOpenSci Champions Program is a 12-month-long program including cohort-based training, project development, and personal mentorship.

Champions will participate in:

* A cohort-based training on:

    - how to plan and facilitate engaging and inclusive workshops to support participants’ success,
    - knowledge of various channels through which new members can engage in and contribute to rOpenSci and R projects, and
    - technical skills in software development and review.

* Personal and group mentorship:

    - A series of meetings with mentors;
    - Quarterly cohort calls.

## What are the benefits of taking part? 

{{< figure src = "/images/volunteer/peep5.png" width = "300" alt = "Person holding a butterfly nest." class = "pull-left" >}}

By participating in this program, you can help rOpenSci and your community grow. In addition to training and personal mentoring, some other benefits you will receive are:

* Learn and practice organizing and teaching workshops;

> * Learn and practice reviewer, and developer skills;

Reviewer: ```suggestion
* Learn and practice reviewer and developer skills;
```

Champions are expected to _complete training and attend meetings with their mentors and cohorts_. After training, Champions will develop their own project. Projects may involve building a new package or going through the peer review process as an author or a reviewer.

Champions will also choose and perform several activities, such as running recruitment activities, giving presentations, writing blog posts, and organizing peer groups in their communities.

It is expected that Champions will promote participation in peer review and contribution to rOpenSci open source software, infrastructure, and documentation. They will also gather feedback on where rOpenSci’s current programming might be improved to meet the needs of all who wish to participate.

The estimated time commitment for each Champion is, on average, __16 hours per month__  (including trainings, cohort events, mentoring meetings, package development or review process, and preparing for and hosting events). The total time commitment for each Champion will vary based on the path and activities they choose to do during the program.

## Frequently Asked Questions

> ### I don't have my own package to work on, can I apply with other people's package?

Reviewer: ```suggestion
### I don't have my own package to work on, can I apply with another's package?
```

> If the package already exists, the possible project is to send it for review.  In that case the requirements are:

Reviewer: ```suggestion
If the package already exists, it is possible to participate by sending the package in for Peer-Review. In this case the requirements would be:
```

> 1) Have written permission (can be an email) from the authors or maintainers of the package.
> 2) Commit to maintain the package for two years.

Reviewer: ```suggestion
1) Written permission (can be an email) from the authors or maintainers of the package for you to take over maintenance and submit the package for review.
2) A commitment from you to maintain the package for at least two years.
```

### Does the program have a cost for participants?

> No, the program it is free.  The champions will get a small stipend for being part of the program after they complete it.

Reviewer: ```suggestion
No, the program it is free.  The champions will get a small stipend for being part of the program after completion.
```

> ### Can we apply it as a team?

Reviewer: ```suggestion
### Can we apply as a team?
```

> No in this call.  You can pick some of the member of the team and they can apply with your project. 

Reviewer: ```suggestion
Not in this call.  You can pick a team member and they can apply with the project. 
```

> Please note that if the person has not contributed to the package previously, you need to accompany the application with a note that he/she/they is part of the team and has permission from the authors and maintainers to work on the package.

Reviewer: ```suggestion
Please note that if the person has not contributed to the package previously, they will need to submit a note that they are part of the package team and have permission from the authors and maintainers to work on the package.
```

### What does it mean to belong to a historically and systematically excluded group?

To belong to a historically and systematically excluded group means being part of a community or identity that has faced long-standing patterns of marginalization, discrimination, and disadvantage. It encompasses groups that have been historically oppressed, discriminated against, or denied equal opportunities based on various factors such as race, ethnicity, gender, sexual orientation, religion, disability, socioeconomic status, geographic location, or any combination thereof.

### Can I propose more than one package?

> The time involved in the program only allows you to work on one package. To help you decide which one to propose in the application, check that the package is in [rOpenSci peer-review scope](https://devguide.ropensci.org/policies.html?q=scope#aims-and-scope) or [rOpenSci stat peer review scope](https://ropensci.org/stat-software-review/).

Reviewer: ```suggestion
The time involved in the program only allows you to work on one package. To help you decide which one to propose in the application, check that the package is within [rOpenSci Peer-review scope](https://devguide.ropensci.org/policies.html?q=scope#aims-and-scope) or [rOpenSci Stats Peer-review scope](https://ropensci.org/stat-software-review/).
```

> ### Can I apply for a project type and then change it?

Reviewer: ```suggestion
### Can I change the type of project I want to do?
```

No. We seek to have a balance in the number of each type of project and we assign mentors also according to the type of project.    

### What does a solid idea mean for a package? What is the appropriate scope? 

> A solid idea refers to a well-conceived concept for the package that you can work on inmeadiately. We don't expect project to explore ideas of packages and we don't provide ideas for new packages. 

Reviewer: ```suggestion
A solid idea refers to a well-conceived concept for the package that you can start work on immediately. After the program starts there will not be enough time to explore different ideas for packages and we don't provide ideas for new packages. 
```

> This are [rOpenSci Review](https://devguide.ropensci.org/policies.html?q=scope#aims-and-scope) and [Stats Review](https://ropensci.org/stat-software-review/) scopes for packages. We expect projects related with research software. 

Reviewer: ```suggestion
These are [rOpenSci Peer-Review](https://devguide.ropensci.org/policies.html?q=scope#aims-and-scope) and [Stats Peer-Review](https://ropensci.org/stat-software-review/) scopes for packages. We expect projects related with research software. 
```
